src/routes/+page.server.ts

```diff
@@ -26,6 +26,27 @@ interface MostFrequentlyServedAsMinisterPolitician extends ComponentProps<StatCa
 	cabinetTerms: number[];
 }
 
+const mpTermDisplayName = (mp: any) => {
```

Comment:
Better use verb for function name, so it will be easier to distinguish between variable (noun) or function (verb). E.g. getMpTermDisplayName